src/lib/y2partitioner/dialogs/partition_table_type.rb

```diff
@@ -0,0 +1,91 @@
+require "yast"
+require "cwm/dialog"
+require "cwm/common_widgets"
+
+module Y2Partitioner
+  module Dialogs
+    # Select a partition table type (GPT, MS-DOS, DASD)
+    # Part of {Actions::CreatePartitionTable}.
+    class PartitionTableType < CWM::Dialog
+      # Choose partition table type
+      class TypeChoice < CWM::RadioButtons
+        # @param controller [Actions::Controllers::PartitionTable]
+        def initialize(controller)
+          textdomain "storage"
+
+          @controller = controller
+          @supported_types = controller.possible_partition_table_types.map(&:to_s)
+          log.info("Supported partition table types: #{@supported_types}")
+        end
+
+        # @macro seeAbstractWidget
+        def label
+          _("New Partition Table Type")
```

Comment:
Yes, but that's wrong; a radio box label should not contain instructions like "select...", only a headline what it is to select from. This is similar to input fields where we also only label them with "user name:" and not like "enter user name:". Keeping it concise helps keeping the textual clutter away from the user. Remember that users have to read that stuff over and over again, and it is obvious what the user should do here: enter something or select something or whatever. We also don't label buttons with "Press Next to continue" etc. because there are visual cues what a widget is, and that's just the same with radio boxes or input fields.

Flooding the user with pointless text is bad UI design.